mon-pix/app/components/certification-starter.js

```diff
@@ -46,8 +46,10 @@ export default Component.extend({
         this.getCurrentCertificationCourse().deleteRecord();
         if (err.errors && err.errors[0] && err.errors[0].status === '404') {
           this.set('errorMessage', 'Ce code n’existe pas ou n’est plus valide.');
+        } else if (err.errors && err.errors[0] && err.errors[0].status === '412') {
+          this.set('errorMessage', 'La session de certification n\'est plus accessible.');
```

Comment:
On devrait pouvoir faire ça:
```suggestion
        } else if (err.errors?[0]?.status === '412') {
          this.set('errorMessage', 'La session de certification n\'est plus accessible.');
```

Reply:
Mon linter m'engueule avec ta version, je suis pourtant bien en config linter 14.16 de node :/